CHANGELOG.md

```diff
@@ -1,5 +1,103 @@
 # Changelog
 
+## 0.3.0
+
+The uv [project](https://docs.astral.sh/uv/guides/projects/),
```

Comment:
"This release introduces the...", maybe?

Reply:
Tweaked